editor/store/test/selectors.js

```diff
@@ -3397,5 +3398,17 @@ describe( 'selectors', () => {
 				[ 'core/block2' ]
 			);
 		} );
+
+		it( 'should return false if no autosave is available', () => {
+			const state = {};
```

Comment:
Is it possible for `hasAutosave` to be called with a state shape of `{}` in standard usage?

With an initial state, `state` would at least be `{ autosave: null }`.

Raising because there's a chance this test is not well-covering the implementation if, for example, the implementation was:

```
hasAutosave = ( state ) => undefined !== state.autosave
```

Where this test case would pass, but the selector itself would produce the wrong result on the initial state.